src/main/java/com/es/phoneshop/model/cart/Cart.java

```diff
@@ -1,5 +1,7 @@
 package com.es.phoneshop.model.cart;
 
+import com.es.phoneshop.model.cart.cart_item.CartItem;
```

Comment:
Underscores are usually not used in package names: https://stackoverflow.com/questions/3179216/what-is-the-convention-for-word-separator-in-java-package-names